lib/codit/outbreak.py

```diff
@@ -1,11 +1,30 @@
 import pandas as pd
 import logging
+import numpy as np
+import math
+import matplotlib.pyplot as plt
+# according to Leeds accommodations' coordinates distribution's 'lon':'lat' ratio, the unit is inch, real size would fit the distribution
+plt.rcParams["figure.figsize"] = [12, 5]
```

Comment:
Don't think that setting the figsize should be in the main flow of the module

Reply:
I will put it in another module during init()